Assignment6_docker/docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+version: "3.8"
+services:
+  bda696_mariadb:
+    image: lscr.io/linuxserver/mariadb
+    container_name: bda696_mariadb
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - MYSQL_ROOT_PASSWORD=abc123 # pragma: allowlist secret
+      - TZ=UTC
+      - MYSQL_USER=root
+      - MYSQL_PASSWORD=abc123 # pragma: allowlist secret
+    volumes:
+      - mariadb-data:/config
+    ports:
+      - "5000:3306"
+    restart: unless-stopped
+
+  bda696_mysql_service:
```

Comment:
This container is just to build-up and run the SQL query to get the rolling avg, right? It's not really a server, so the name of this container may better be like "bda696_hw6".